Sources/MealzNavModuleIOS/NavCoordinators/RecipeCard/MealzStandaloneRecipeCardSwiftUI.swift

```diff
@@ -76,18 +76,19 @@ public struct MealzStandaloneRecipeCardSwiftUI: UIViewControllerRepresentable {
         )
     }
     
-    public func makeUIViewController(context: Context) -> UINavigationController {
+    public func makeUIViewController(context: Context) -> UIViewController {
         if let criteria {
             coordinator.showRecipeCard(criteria: criteria)
         } else if let recipe {
             coordinator.showRecipeCard(recipe: recipe)
         } else if let recipeId {
             coordinator.showRecipeCard(recipeId: recipeId)
         }
-        return coordinator.navigationController
+        
+        return coordinator.navigationController.viewControllers.first!
```

Comment:
should we be concerned about this force cast potentially crashing? Is there a way to return nothing/nil in that case?

Reply:
Corrigé